samples/SmokeTest/Source/Assets/SmokeTest/MainGui.cs

```diff
@@ -936,6 +941,45 @@ private void ShowPopupGravityUi()
             }
         }
 
+        private void ShowPermissionsUi()
+        {
+            DrawStatus();
+            DrawTitle("PopupGravity Ui");
+
+            if (GUI.Button(CalcGrid(0, 1), "Has Drive - AppData"))
+            {
+                Status = "Drive - AppData Permission " + PlayGamesPlatform.Instance.HasPermission(
+                             /* scope= */ "https://www.googleapis.com/auth/drive.appdata");
+            }
```

Comment:
We shouldn't use alone drive scopes in sample.

Reply:
changed to email